src/bucket.js

```diff
@@ -1960,6 +2032,51 @@ class Bucket extends common.ServiceObject {
     );
   }
 
+  /**
+   * Lock all objects contained in the bucket, based on their creation time. Any
+   * attempt to overwrite or delete objects younger than the retention period
+   * will result in a `PERMISSION_DENIED` error.
+   *
+   * An unlocked retention policy can be modified or removed from the bucket via
+   * {@link File#removeRetentionPeriod} and {@link File#setRetentionPeriod}. A
+   * locked retention policy cannot be removed or shortened in duration for the
+   * lifetime of the bucket. Attempting to remove or decrease period of a locked
+   * retention policy will result in a `PERMISSION_DENIED` error.
```

Comment:
Retention policy can be increased when the retention policy is locked.

Reply:
This is actually a copy/paste of what is in the REST docs: https://www.googleapis.com/discovery/v1/apis/storage/v1/rest

I updated ours to say "You can still increase the policy." at the end, although I think it is implied by the previous mentions of "...cannot be removed or shortened..." and "Attempting to remove or decrease...".

Just mentioning in case you want to make the option to increase the policy more pronounced in the REST docs as well.

Reply:
Thanks for the feedback, I'll forward this!